pkgs/by-name/rl/rlama/package.nix

```diff
@@ -37,25 +38,41 @@ buildGoModule rec {
     makeWrapper
   ];
 
+  # Run only unit tests for core packages; skip e2e tests that require Ollama
+  checkPhase = ''
+    go test -v ./internal/domain/... ./pkg/vector/... ./internal/repository/...
+  '';
+
   postInstall = ''
     wrapProgram $out/bin/rlama \
       --prefix PATH : ${
         lib.makeBinPath [
           poppler-utils
           tesseract
           catdoc
+          unrtf
           python3Packages.pdfminer-six
           python3Packages.docx2txt
           python3Packages.xlsx2csv
+          python3Packages.torch
+          python3Packages.transformers
         ]
       }
   '';
 
   nativeInstallCheckInputs = [
-    versionCheckHook
+    writableTmpDirAsHomeHook
   ];
-  versionCheckProgramArg = "--version";
+
   doInstallCheck = true;
+  installCheckPhase = ''
```

Comment:
Why not using versionCheckHook ?

Reply:
Because writableTmpDirAsHomeHook need to be active before versionCheckHook

Reply:
You can perhaps use https://github.com/NixOS/nixpkgs/pull/403971 ?

Reply:
Im not think this is good way for this app.
But i dont know how to fix this for use `versionCheckHook`

Reply:
When try : 

```
  nativeInstallCheckInputs = [
    writableTmpDirAsHomeHook
    versionCheckHook
  ];
  versionCheckProgramArg = "--version";
  versionCheckDontIgnoreEnvironment = true;
  doInstallCheck = true;
  ```
  
Have : 
_panic: Could not determine user home directory: $HOME is not defined_

Reply:
Right, this is only on staging at the moment... Sad :(

Reply:
When this will land: https://nixpkgs-tracker.ocfox.me/?pr=403971

Reply:
Im using https://nixpk.gs/pr-tracker.html